lsp-dart.el

```diff
@@ -60,6 +60,11 @@
   :type '(repeat string)
   :group 'lsp-dart)
 
+(defcustom lsp-dart-test-extra-args '()
+  "Extra arguments to be passed to test command (e.g: --no-sound-null-safety)."
+  :type '(repeat string)
+  :group 'lsp-dart)
+
```

Comment:
Since this defcustom is specific for `lsp-dart-test` could you move it to `lsp-dart-test.el`?

Reply:
Done